tests/select/test_select_pv_systems_without_output_wholeday.py

```diff
@@ -0,0 +1,45 @@
+import logging
```

Comment:
Great job on these! I also recommend having the tests with the PVOutput datapipe `pvoutput_datapipe` to check it works for both that we use